frontend/src/index.css

```diff
@@ -14,6 +14,28 @@
   body {
     @apply bg-background text-foreground;
   }
+
+  :root {
```

Comment:
not important, but do we need to update the other themes later?

Reply:
I think the current style is okay for all the themes we have (that's why I added it to the base style), but for future themes that would definitely make sense :+1: